### PR TITLE
Dialogs - styling, html, mock functions

### PR DESCRIPTION
**Pull request by BartoszBilejczyk**

### CODE REVIEW

File: src/app/common/dialog/add-funds-dialog/add-funds-dialog.component.html [hunk 1]
```diff
@@ -0,0 +1,86 @@
...
+        value="0xe99356bde974bbe08721d77712168fa070aa8da4"
+        disabled
+      >
+      <span class="input-valid"></span>
+    </div>
+    <p
+      class="
+          adsh-copy
+          adsh-copy--large"
+    >
```
Comment (KonradFak) [inline src/app/common/dialog/add-funds-dialog/add-funds-dialog.component.html]: Too large indents

File: src/app/common/dialog/add-funds-dialog/add-funds-dialog.component.html [hunk 2]
```diff
@@ -0,0 +1,86 @@
...
+        value="User 0291"
+        disabled
+      >
+      <span class="input-valid"></span>
+    </div>
+    <p
+      class="
+          adsh-copy
+          adsh-copy--large"
+    >
```
Comment (KonradFak) [inline src/app/common/dialog/add-funds-dialog/add-funds-dialog.component.html]: Too large indents

File: src/app/common/dialog/add-funds-dialog/add-funds-dialog.component.html [hunk 3]
```diff
@@ -0,0 +1,86 @@
...
+      >
+        Our Ethereum Address
+      </label>
+      <input
+        id="ethereum-address"
+        class="
+          adsh-form-input
+          adsh-form-input--full"
+        name="ethereum-address"
+        value="0xe99356bde974bbe08721d77712168fa070aa8da4"
```
Comment (KonradFak) [inline src/app/common/dialog/add-funds-dialog/add-funds-dialog.component.html]: Should be some value got from backend, and maybe saved in store when app is started

File: src/app/common/dialog/change-address-dialog/change-address-dialog.component.html [hunk 4]
```diff
@@ -0,0 +1,97 @@
...
+        >
+          Our Ethereum Address
+        </label>
+        <input
+          id="ethereum-address"
+          class="
+            adsh-form-input
+            adsh-form-input--full"
+          name="ethereum-address"
+          value="0xe99356bde974bbe08721d77712168fa070aa8da4"
```
Comment (KonradFak) [inline src/app/common/dialog/change-address-dialog/change-address-dialog.component.html]: See comment about backend

File: src/app/models/initial-state/user.ts [hunk 5]
```diff
@@ -6,5 +6,7 @@ export const userInitialState: User = {
   isAdvertiser: true,
   isPublisher: true,
   isAdmin: false,
+  userAutomaticWithdrawPeriod: 'month',
```
Comment (KonradFak) [inline src/app/models/initial-state/user.ts]: Should be integer based on `withdrawPeriodsEnum `

File: src/app/common/dialog/add-funds-dialog/add-funds-dialog.component.html [hunk 6]
```diff
@@ -0,0 +1,86 @@
...
+        Your MEMO
+      </label>
+      <input
+        id="memo"
+        class="
+          adsh-form-input
+          adsh-form-input--full"
+        type="url"
+        name="memo"
+        value="User 0291"
```
Comment (KonradFak) [inline src/app/common/dialog/add-funds-dialog/add-funds-dialog.component.html]: Shouldn't this also be something that user can change?

File: src/app/common/dialog/change-address-dialog/change-address-dialog.component.ts [hunk 7]
```diff
@@ -0,0 +1,69 @@
...
+
+  createForm() {
+    this.changeWithdrawAddressForm = new FormGroup({
+      address: new FormControl('', [
+        Validators.required,
+        Validators.minLength(42),
+        Validators.maxLength(42)
+      ])
+    });
+
```
Comment (KonradFak) [inline src/app/common/dialog/change-address-dialog/change-address-dialog.component.ts]: unnecessary new line

File: src/app/common/dialog/change-address-dialog/change-address-dialog.component.ts [hunk 8]
```diff
@@ -0,0 +1,69 @@
...
+    if (!this.changeWithdrawAddressForm.valid) {
+      return;
+    }
+
+    this.isFormBeingSubmitted = true;
+
+    const changeWithdrawAddressSubscription = this.settingsService
+      .changeWithdrawAddress(this.changeWithdrawAddressForm.value.address)
+      .subscribe(() => {
+        this.dialogRef.close();
```
Comment (KonradFak) [inline src/app/common/dialog/change-address-dialog/change-address-dialog.component.ts]: Also you should update data in local storage and sotore

File: src/app/common/dialog/change-automatic-withdraw-dialog/change-automatic-withdraw-dialog.component.ts [hunk 9]
```diff
@@ -0,0 +1,79 @@
...
+@Component({
+  selector: 'app-change-automatic-withdraw-dialog',
+  templateUrl: './change-automatic-withdraw-dialog.component.html',
+  styleUrls: ['./change-automatic-withdraw-dialog.component.scss']
+})
+export class ChangeAutomaticWithdrawDialogComponent extends HandleSubscription implements OnInit {
+  automaticWithdrawForm: FormGroup;
+
+  periods = enumToArray(withdrawPeriodsEnum);
+  amounts = [100, 200, 300, 400, 500, 600, 700, 800, 900, 1000];
```
Comment (KonradFak) [inline src/app/common/dialog/change-automatic-withdraw-dialog/change-automatic-withdraw-dialog.component.ts]: Maybe this amounts should also be taken from settings or some enum?

File: src/app/common/dialog/change-automatic-withdraw-dialog/change-automatic-withdraw-dialog.component.ts [hunk 10]
```diff
@@ -0,0 +1,79 @@
...
+  ) {
+    super(null);
+  }
+
+  ngOnInit() {
+    this.createForm();
+
+    const currentPeriodSubscription = this.store.select('state', 'user', 'data', 'userAutomaticWithdrawPeriod')
+      .subscribe((currentPeriod: number) => {
+      this.currentPeriod = this.periods[currentPeriod];
```
Comment (KonradFak) [inline src/app/common/dialog/change-automatic-withdraw-dialog/change-automatic-withdraw-dialog.component.ts]: Fix indentation

File: src/app/common/dialog/change-automatic-withdraw-dialog/change-automatic-withdraw-dialog.component.ts [hunk 11]
```diff
@@ -0,0 +1,79 @@
...
+    });
+
+    const currentAmountSubscription = this.store.select('state', 'user', 'data', 'userAutomaticWithdrawAmount')
+      .subscribe((currentAmount: number) => {
+      this.currentAmount = currentAmount;
```
Comment (KonradFak) [inline src/app/common/dialog/change-automatic-withdraw-dialog/change-automatic-withdraw-dialog.component.ts]: Fix indentation

File: src/app/common/dialog/withdraw-funds-dialog/withdraw-funds-dialog.component.html [hunk 12]
```diff
@@ -0,0 +1,196 @@
...
+      </p>
+
+      <div class="total-funds">
+        <p class="adsh-form-label">
+          Total Funds
+        </p>
+        <h1
+          class="
+          adsh-heading
+          adsh-heading--primary"
```
Comment (KonradFak) [inline src/app/common/dialog/withdraw-funds-dialog/withdraw-funds-dialog.component.html]: Fix indentation

File: src/app/common/dialog/withdraw-funds-dialog/withdraw-funds-dialog.component.html [hunk 13]
```diff
@@ -0,0 +1,196 @@
...
+      </div>
+
+      <div
+        class="
+          calculations
+          adsh-copy
+          adsh-copy--x-dark
+          adsh-copy--normal"
+      >
+        <p>Transaction Fee:
```
Comment (KonradFak) [inline src/app/common/dialog/withdraw-funds-dialog/withdraw-funds-dialog.component.html]: Text value should be in new line

File: src/app/models/initial-state/user.ts [hunk 14]
```diff
@@ -6,5 +6,8 @@ export const userInitialState: User = {
   isAdvertiser: true,
   isPublisher: true,
   isAdmin: false,
+  userEthAddress: '0xe99356bde974bbe08721d77712168fa070aa8da4',
```
Comment (KonradFak) [inline src/app/models/initial-state/user.ts]: should be some empty value i think

File: src/app/common/dialog/withdraw-funds-dialog/withdraw-funds-dialog.component.html (hunk 13 above)
Comment (BartoszBilejczyk) [reply src/app/common/dialog/withdraw-funds-dialog/withdraw-funds-dialog.component.html]: done

File: src/app/models/initial-state/user.ts (hunk 14 above)
Comment (BartoszBilejczyk) [reply src/app/models/initial-state/user.ts]: done

File: src/app/common/dialog/withdraw-funds-dialog/withdraw-funds-dialog.component.html (hunk 12 above)
Comment (BartoszBilejczyk) [reply src/app/common/dialog/withdraw-funds-dialog/withdraw-funds-dialog.component.html]: done

File: src/app/common/dialog/change-automatic-withdraw-dialog/change-automatic-withdraw-dialog.component.ts (hunk 11 above)
Comment (BartoszBilejczyk) [reply src/app/common/dialog/change-automatic-withdraw-dialog/change-automatic-withdraw-dialog.component.ts]: done

File: src/app/common/dialog/change-automatic-withdraw-dialog/change-automatic-withdraw-dialog.component.ts (hunk 10 above)
Comment (BartoszBilejczyk) [reply src/app/common/dialog/change-automatic-withdraw-dialog/change-automatic-withdraw-dialog.component.ts]: done

File: src/app/common/dialog/change-address-dialog/change-address-dialog.component.ts (hunk 7 above)
Comment (BartoszBilejczyk) [reply src/app/common/dialog/change-address-dialog/change-address-dialog.component.ts]: done

**Pull request merged**